src/extras/helpers/PolarGridHelper.js

```diff
@@ -0,0 +1,94 @@
+import { LineSegments } from '../../objects/LineSegments';
+import { VertexColors } from '../../constants';
+import { LineBasicMaterial } from '../../materials/LineBasicMaterial';
+import { Float32Attribute } from '../../core/BufferAttribute';
+import { BufferGeometry } from '../../core/BufferGeometry';
+import { Color } from '../../math/Color';
+import { _Math as Math } from '../../math/Math';
```

Comment:
`_Math` was the reference to `three.js` internal math lib. In `PolarGridHelper` we just use the plain `Math` object of JavaScript (see http://www.w3schools.com/js/js_math.asp)